Code/Framework/AzToolsFramework/AzToolsFramework/AssetBrowser/Views/AssetBrowserTableView.cpp

```diff
@@ -146,6 +143,20 @@ namespace AzToolsFramework
 
         void AssetBrowserTableView::OnAssetBrowserComponentReady()
         {
+            horizontalHeader()->setStretchLastSection(true);
+            setColumnWidth(0, aznumeric_cast<int>(parentWidget()->width() * 0.5f));
+            horizontalHeader()->setMinimumSectionSize(aznumeric_cast<int>(parentWidget()->width() * 0.25f));
+            horizontalHeader()->setMaximumSectionSize(aznumeric_cast<int>(parentWidget()->width() * 0.75f));
+            horizontalHeader()->setSectionResizeMode(0, QHeaderView::ResizeMode::Interactive);
+            horizontalHeader()->setSectionResizeMode(1, QHeaderView::ResizeMode::Interactive);
+        }
+
+        void AssetBrowserTableView::UpdateSizeSlot(int newWidth)
+        {
+           // int parentWidth = parentWidget()->geometry().width();
+            setColumnWidth(0, aznumeric_cast<int>(newWidth * 0.5f));
+            horizontalHeader()->setMinimumSectionSize(aznumeric_cast<int>(newWidth * 0.25f));
+            horizontalHeader()->setMaximumSectionSize(aznumeric_cast<int>(newWidth * 0.75f));
```

Comment:
update: As these value are repeated they should definitely be named constants (maybe the 0.5f above too)